install-panel.sh

```diff
@@ -1040,12 +938,11 @@ function goodbye {
   echo "* Unofficial add-ons and tips"
   echo "* - Third-party themes, $(hyperlink 'https://github.com/TheFonix/Pterodactyl-Themes')"
   echo "*"
-  echo "* Installation is using $WEBSERVER on $OS"
+  echo "* Installation is using nginx on $OS"
   echo "* Thank you for using this script."
   echo -e "* ${COLOR_RED}Note${COLOR_NC}: If you haven't configured the firewall: 80/443 (HTTP/HTTPS) is required to be open!"
   print_brake 62
 }
 
 # run script
-main
-goodbye
```

Comment:
This is not consistent with the Wings script. Open a PR to edit the Wings script or modify the PR on this one. It doesn't really matter to me, not really a necessary change.

Reply:
Fixed.